plugins/connection/incus.py

```diff
@@ -142,10 +142,14 @@ def _build_command(self, cmd) -> list[str]:
         ]
 
         if getattr(self._shell, "_IS_WINDOWS", False):
-            if (
-                (regex_match := self.powershell_regex_pattern.match(cmd))
-                and (regex_pattern := self.powershell_regex_pattern)
-            ) or ((regex_match := self.cmd_regex_pattern.match(cmd)) and (regex_pattern := self.cmd_regex_pattern)):
+            if regex_match := self.powershell_regex_pattern.match(cmd):
+                regex_pattern = self.powershell_regex_pattern
+            elif regex_match := self.cmd_regex_pattern.match(cmd):
+                regex_pattern = self.cmd_regex_pattern
+            else:
+                regex_match = None
```

Comment:
Actually this can also be removed, since `regex_match` has already been set in the `if` and `elif`.
```suggestion
```
It doesn't hurt to keep it though...